Integrations/integration-SumoLogic.yml

```diff
@@ -215,7 +215,7 @@ script:
             }
 
             var s = search(params.fetchQuery, lastRun.time, now, a2i(params.limit, defaultLimit), 0, params.timeZone, a2i(params.maxTimeout, 180), false,
-                a2i(params.sleepBetweenChecks, defaultSleep));
+                a2i(params.sleepBetweenChecks, defaultSleep), params.fetchRecords);
```

Comment:
why do we need the addition of params.fetchRecords here?
Shouldn't we want this to be `true` for both records and messages?

Reply:
No, when fetching messages, we are still only returning the first (params.limit / defaultLimit) entries. In this case, we can return early from the search.
For aggregate records, we want to iterate over all messages, as otherwise the aggregations will be computed over a partial data set only.